app/components/user-form/component.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { typeOf } from '@ember/utils';
+import { inject as service } from '@ember/service';
+
+export default class UserFormComponent extends Component {
+  @service firebase;
+
+  constructor() {
+    super(...arguments);
+    this.data = {
+      name: null,
+      surname: null,
+      email: null,
+      pswd: null,
+      rpswd: null,
+    };
+  }
+
+  @action
+  cancel() {
+    this.args.changeset.rollback();
+    let inputs = document.querySelectorAll('input');
+    inputs.forEach((input) => (input.value = ''));
+    if (typeOf(this.args.toggleState) === 'function') {
+      this.args.toggleState();
+    }
+  }
+  @action
```

Comment:
In my personal suggestion, good approach is adding empty line between functions. In bigger components you'll see a wall of code, wich is little not readable.